src/components/Navbar/Navbar.css

```diff
@@ -33,9 +34,9 @@
 }
 
 .logo{
-    margin-top: 10px;
+    margin: 10px 0;
```

Comment:
Applying the margin this way would also add `10px` of `margin-bottom`. Is that the intention or you just would like the minify it with `margin: 10px 0 0;`